src/NHibernate/Transaction/AdoNetWithDistributedTransactionFactory.cs

```diff
@@ -1,3 +1,4 @@
+#if !NETSTANDARD2_0
```

Comment:
Having this class enabled makes my failure count go from 67 to 1,676.

Simply calling `new TransactionScope()` causes: `System.Transactions.TransactionAbortedException : The transaction has aborted.`.

Calling `transactionContext.AmbientTransation.EnlistVolatile(transactionContext, EnlistmentOptions.EnlistDuringPrepareRequired);` results in: `System.Transactions.TransactionException : The operation is not valid for the state of the transaction.`.

Calling `System.Transactions.Transaction.Current.EnlistDurable(Guid.NewGuid(), this, EnlistmentOptions.None);` results in: `System.PlatformNotSupportedException : This platform does not support distributed transactions.`.

Maybe it's supported, but it doesn't seem to be working for me.  And this is Windows.  What does a distributed transaction even look like on Linux?

This is the implementation: dotnet/corefx#10089 Maybe they are missing something we need?